src/scripts/actions/UploaderActions.js

```diff
@@ -60,9 +60,7 @@ export const uploadAndTranscode = (file: Object, videoId: string) => (
     videoId = paratii.eth.vids.makeId()
   }
   dispatch(
-    videoFetchSuccess(
-      new VideoRecord({ id: videoId, owner: paratii.config.account.address })
-    )
+    videoFetchSuccess({ id: videoId, owner: paratii.config.account.address })
```

Comment:
I modified the video reducer to take plain objects as `payloads`, so actions should no longer pass in immutable records. The reasoning is so that we can pass in plain JS objects to our record constructors and the constructors will worry about "automatically" converting all of the properties into immutable structures.